google_analytics/notebooks/load.ipynb

```diff
@@ -68,20 +61,22 @@
     "    print(\"Generating date columns...\")\n",
     "    merged['WoY'] = merged['date'].apply(lambda x: x.isocalendar()[1])\n",
     "    merged['month'] = merged['date'].apply(lambda x: x.month)\n",
-    "    merged['quarter_month'] = merged['date'].apply(lambda x: x.day // 8)\n",
+    "    merged['quarterMonth'] = merged['date'].apply(lambda x: x.day // 8)\n",
     "    merged['weekday'] = merged['date'].apply(lambda x: x.weekday())\n",
     "    del merged['date']\n",
     "\n",
     "    format_time = lambda t: time.strftime('%Y-%m-%d %H:%M:%S', time.localtime(t))\n",
-    "    merged['visit_hour'] = pd.to_datetime(merged['visitStartTime'].apply(format_time)).apply(lambda t: t.hour)\n",
+    "    merged['visitHour'] = pd.to_datetime(merged['visitStartTime'].apply(format_time)).apply(lambda t: t.hour)\n",
     "    del merged['visitStartTime']\n",
     "    \n",
-    "    print(\"Encoding columns with pd.factorize()\")\n",
-    "    for col in merged.columns:\n",
-    "        if col in ['fullVisitorId', 'month', 'quarter_month', 'weekday', 'visit_hour', 'WoY']:\n",
-    "            continue\n",
-    "        if merged[col].dtypes in [object, bool]:\n",
-    "            merged[col], indexer = pd.factorize(merged[col])\n",
+    "    print(\"Finding total visits...\")\n",
```

Comment:
I think it's fine, but at the same time not necessary to add this column here. When we aggregate the data per customer, we can/will take the maximum of the visit number as a feature, which would lead to the same result (but might be 'cleaner').